global/meta/permission.yml

```diff
@@ -10,6 +10,9 @@ agenda_item:
     can_manage:
         can_see_internal:
             can_see:
+    can_manage_moderator_notes:
+        can_see_moderator_notes:
```

Comment:
Are you sure, that a user with `agenda_item.can_manage` can not see the moderator notes?

Reply:
Not by default, yes. Discussed with @rrenkert that there might be cases where the permissions for the agenda item and the moderator notes might be required to be split up. If necessary, the user can just give both permissions to every group to archieve the wanted behaviour.